tests/cypress/features/delegates.feature

```diff
@@ -12,24 +12,24 @@ Feature: validator
   #   And transactions count should have value greater than 0
 
   # @basic
-  # Scenario: Should properly display forging details
-  #   When I observe blocksForged
+  # Scenario: Should properly display generating details
```

Comment:
@oskarleonard This file and similarly named files in the `features` folder should be renamed

Reply:
Done